src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days < 7) {
```

Comment:
I wouldn't use else if structure in this case because every if closure has return statement. Use simple if
for example:
If (condition1) { return value1 }
if (condition2) {return value2 }